unified-codes/apps/web/src/components/containers/detail/DetailPropertyListItem.tsx

```diff
@@ -21,33 +23,48 @@ const useStyles = makeStyles((_: ITheme) =>
     root: {
       width: '100%',
     },
+    listItemRoot: { paddingBottom: 0, paddingTop: 0}
   })
 );
 
 interface DetailPropertyListItemProps {
-  description?: string;
   properties?: IProperty[];
+  type?: string;
+  value?: string;
+}
+
+const formatProperty = (type?: string, value?: string) => {
+  const formatted = propertyFormatter(type, value);
+  const { title, url } = formatted;
```

Comment:
Since we are changing the `DetailPropertyListItem` props, I think this stuff can be done in `mapStateToProps`? Then we could add this formatting logic to selectors (`DetailSelectors`)?

Reply:
nice idea!